pkg/oc/clusterup/up.go

```diff
@@ -87,11 +87,11 @@ var (
 )
 
 // NewCmdUp creates a command that starts OpenShift on Docker with reasonable defaults
-func NewCmdUp(name, fullName string, f genericclioptions.RESTClientGetter, out, errout io.Writer, clusterAdd *cobra.Command) *cobra.Command {
+func NewCmdUp(name, fullName string, f genericclioptions.RESTClientGetter, streams genericclioptions.IOStreams, clusterAdd *cobra.Command) *cobra.Command {
 	config := &ClusterUpConfig{
```

Comment:
nit: same thing here - NewClusterUpConfig(), and use io streams directly from its methods rather than passing them in as args

Reply:
See https://github.com/openshift/origin/pull/20232